cli/command/service/opts.go

```diff
@@ -494,6 +496,8 @@ func addServiceFlags(flags *pflag.FlagSet, opts *serviceOptions) {
 	flags.StringVar(&opts.update.onFailure, flagUpdateFailureAction, "pause", `Action on update failure ("pause"|"continue"|"rollback")`)
 	flags.Var(&opts.update.maxFailureRatio, flagUpdateMaxFailureRatio, "Failure rate to tolerate during an update")
 	flags.SetAnnotation(flagUpdateMaxFailureRatio, "version", []string{"1.25"})
+	flags.StringVar(&opts.update.order, flagUpdateOrder, "start-first", `Update order ("start-first"|"stop-first")`)
```

Comment:
Default should be `stop-first`. `stop-first` is usually safe as tasks would not compete for resources, like CPU/mem/TCP ports, while `start-first` is not. 

https://github.com/docker/swarmkit/blob/master/cmd/swarmctl/service/flagparser/flags.go#L39